databuilder/databuilder/extractor/es_base_extractor.py

```diff
@@ -2,22 +2,37 @@
 # SPDX-License-Identifier: Apache-2.0
 import abc
 from typing import (
-    Any, Dict, Iterator, Optional, Union,
+    Any, Dict, Iterator, Optional, Union, List,
 )
 
 from pyhocon import ConfigTree
 
 from databuilder.extractor.base_extractor import Extractor
+from databuilder.models.table_metadata import ColumnMetadata
 
 
 class ElasticsearchBaseExtractor(Extractor):
     """
     Extractor to extract index metadata from Elasticsearch
+
+    By default, the extractor does not extract nested columns. Set ELASTICSEARCH_EXTRACT_NESTED_COLUMNS conf to True
```

Comment:
Can we make it default and only behavior? 

Btw. nested field is sth else so I wouldn't confuse the terminology (see my comment in the PR thread) https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html#nested

Reply:
Thanks for the clarification on nested fields! I agree with making it default and only behavior. I updated the code and README.